packages/contracts/test/voting/addresslist-voting.ts

```diff
@@ -1226,7 +1214,11 @@ describe('AddresslistVoting', function () {
         votingSettings.supportThreshold = pct16(0);
         votingSettings.minParticipation = pct16(0);
 
-        await voting.initialize(dao.address, votingSettings, addresslist(10));
+        await voting.initialize(
+          dao.address,
+          votingSettings,
+          [0, 1, 2, 3, 4, 5, 6, 7, 8, 9].map(i => signers[i].address)
```

Comment:
using `slice(...)` with `map(...)` would probably be cleaner: `signers.slice(0, 9).map(s => s.address)`

Reply:
Fixed in 26e42329bc5434e96fd75c0bb61139616678b262